bot/internal/review/review.go

```diff
@@ -438,20 +438,17 @@ func (r *Assignments) CheckInternal(e *env.Environment, reviews []github.Review,
 	switch {
 	case changes.Docs && changes.Code:
 		log.Printf("Check: Found docs and code changes.")
-		if err := r.checkInternalDocsReviews(e, reviews, files); err != nil {
-			return trace.Wrap(err)
-		}
-		if err := r.checkInternalCodeReviews(e, changes, reviews); err != nil {
+		if err := r.checkInternalReviews(e, changes, reviews, files); err != nil {
```

Comment:
nbd - code is easy to follow - but this same call is repeated in 3 of the 4 cases. I did have to look closely to see that the first three cases made the same call. One option would be to just log in the 3 cases, return from the 4th admin check, then make this call below the switch.